roles/wazuh/ansible-wazuh-manager/tasks/Debian.yml

```diff
@@ -117,7 +117,7 @@
 
 - include_tasks: "installation_from_custom_packages.yml"
   when:
-    - wazuh_custom_packages_installation_manager_enabled or not wazuh_custom_packages_installation_manager_enabled
+    - wazuh_custom_packages_installation_manager_enabled or wazuh_custom_packages_installation_api_enabled
 
 - name: Debian/Ubuntu | Install wazuh-api
   apt:
```

Comment:
LGTM but please provide logs of succesful installation with:

- Manager enabled and API enabled
- Manager disabled and API enabled

Thanks!

Reply:
I manually tested the combinatory possibilities while implementing #376, here are the results

```

MANAGER YES | API YES


Gathering Facts ------------------------------------------------------------------------------------------------------------------------------- 16.27s
../roles/wazuh/ansible-wazuh-manager : Install Wazuh Manager from .rpm packages | dnf --------------------------------------------------------- 80.79s
../roles/wazuh/ansible-wazuh-manager : restart wazuh-manager ---------------------------------------------------------------------------------- 10.79s
../roles/wazuh/ansible-wazuh-manager : Install Wazuh API from .rpm packages | dnf ---------------------------------------------------------- 8.78s
../roles/wazuh/ansible-wazuh-manager : Install dependencies ------------------------------------------------------------------------------------ 1.25s
../roles/wazuh/ansible-wazuh-manager : Install expect ------------------------------------------------------------------------------------------ 1.19s
../roles/wazuh/ansible-wazuh-manager : Ensure Wazuh Manager, wazuh API service is started and enabled ------------------------------------------ 1.11s
../roles/wazuh/ansible-wazuh-manager : restart wazuh-api --------------------------------------------------------------------------------------- 1.02s
../roles/wazuh/ansible-wazuh-manager : Installing the local_rules.xml (default local_rules.xml) ------------------------------------------------ 0.97s
../roles/wazuh/ansible-wazuh-manager : Installing NodeJS --------------------------------------------------------------------------------------- 0.93s
../roles/wazuh/ansible-wazuh-manager : RedHat/CentOS/Fedora | Install openscap ----------------------------------------------------------------- 0.92s
../roles/wazuh/ansible-wazuh-manager : Configure ossec.conf ------------------------------------------------------------------------------------ 0.69s
../roles/wazuh/ansible-wazuh-manager : Adding local rules files -------------------------------------------------------------------------------- 0.67s
../roles/wazuh/ansible-wazuh-manager : Installing the local_decoder.xml ------------------------------------------------------------------------ 0.63s
../roles/wazuh/ansible-wazuh-manager : Installing the config.js (api configuration) ------------------------------------------------------------ 0.63s
../roles/wazuh/ansible-wazuh-manager : Installing the local_internal_options.conf -------------------------------------------------------------- 0.61s
../roles/wazuh/ansible-wazuh-manager : Wazuh-API User ------------------------------------------------------------------------------------------ 0.61s
../roles/wazuh/ansible-wazuh-manager : Adding local decoders files ----------------------------------------------------------------------------- 0.59s
../roles/wazuh/ansible-wazuh-manager : Check if NodeJS service exists -------------------------------------------------------------------------- 0.47s
../roles/wazuh/ansible-wazuh-manager : Verifying for old systemd authd service ----------------------------------------------------------------- 0.46s


MANAGER NO | API YES

Gathering Facts ---------------------------------------------------------------------------------------------------- 17.19s
../roles/wazuh/ansible-wazuh-manager : CentOS/RedHat/Amazon | Install wazuh-manager -------------------------------- 85.22s
../roles/wazuh/ansible-wazuh-manager : RedHat/CentOS/Fedora | Install openscap -------------------------------------- 4.93s
../roles/wazuh/ansible-wazuh-manager : Install Wazuh API from .rpm packages | dnf ----------------------------------- 4.91s
../roles/wazuh/ansible-wazuh-manager : Install dependencies --------------------------------------------------------- 1.30s
../roles/wazuh/ansible-wazuh-manager : Installing NodeJS ------------------------------------------------------------ 0.91s
../roles/wazuh/ansible-wazuh-manager : Check if NodeJS service exists ----------------------------------------------- 0.41s
../roles/wazuh/ansible-wazuh-manager : RedHat/CentOS/Fedora | Install Wazuh repo ------------------------------------ 0.36s
../roles/wazuh/ansible-wazuh-manager : include_tasks ---------------------------------------------------------------- 0.05s
../roles/wazuh/ansible-wazuh-manager : include_tasks ---------------------------------------------------------------- 0.04s
../roles/wazuh/ansible-wazuh-manager : RedHat 6 | Enabling Red Hat Software Collections (RHSCL) --------------------- 0.02s
../roles/wazuh/ansible-wazuh-manager : RedHat/CentOS/Fedora | Install python-cryptography module -------------------- 0.02s
../roles/wazuh/ansible-wazuh-manager : RedHat/CentOS 5 | Install Wazuh repo ----------------------------------------- 0.02s
../roles/wazuh/ansible-wazuh-manager : Set Distribution CIS filename for RHEL6/CentOS-6 ----------------------------- 0.02s
../roles/wazuh/ansible-wazuh-manager : Install Wazuh Manager from .rpm packages | yum ------------------------------- 0.02s
../roles/wazuh/ansible-wazuh-manager : Install Wazuh API from .rpm packages | yum ----------------------------------- 0.02s
../roles/wazuh/ansible-wazuh-manager : Install Wazuh Manager from .rpm packages | dnf ------------------------------- 0.02s
../roles/wazuh/ansible-wazuh-manager : include_tasks ---------------------------------------------------------------- 0.02s
../roles/wazuh/ansible-wazuh-manager : CentOS/RedHat 6 | Install Python 2.7 ----------------------------------------- 0.02s
../roles/wazuh/ansible-wazuh-manager : Set Distribution CIS filename for RHEL7/CentOS-7 ----------------------------- 0.02s


MANAGER NO | API NO

Gathering Facts ---------------------------------------------------------------------------------------------------- 21.31s
../roles/wazuh/ansible-wazuh-manager : CentOS/RedHat/Amazon | Install wazuh-manager -------------------------------- 51.53s
../roles/wazuh/ansible-wazuh-manager : CentOS/RedHat/Amazon | Install wazuh-api ------------------------------------- 6.77s
../roles/wazuh/ansible-wazuh-manager : restart wazuh-manager -------------------------------------------------------- 5.92s
../roles/wazuh/ansible-wazuh-manager : Install dependencies --------------------------------------------------------- 1.25s
../roles/wazuh/ansible-wazuh-manager : Installing NodeJS ------------------------------------------------------------ 1.12s
../roles/wazuh/ansible-wazuh-manager : Ensure Wazuh Manager, wazuh API service is started and enabled --------------- 1.05s
../roles/wazuh/ansible-wazuh-manager : RedHat/CentOS/Fedora | Install openscap -------------------------------------- 0.98s
../roles/wazuh/ansible-wazuh-manager : Install expect --------------------------------------------------------------- 0.96s
../roles/wazuh/ansible-wazuh-manager : Configure ossec.conf --------------------------------------------------------- 0.63s
../roles/wazuh/ansible-wazuh-manager : Installing the local_rules.xml (default local_rules.xml) --------------------- 0.63s
../roles/wazuh/ansible-wazuh-manager : restart wazuh-api ------------------------------------------------------------ 0.59s
../roles/wazuh/ansible-wazuh-manager : Wazuh-API User --------------------------------------------------------------- 0.58s
../roles/wazuh/ansible-wazuh-manager : Installing the local_internal_options.conf ----------------------------------- 0.56s
../roles/wazuh/ansible-wazuh-manager : Installing the config.js (api configuration) --------------------------------- 0.56s
../roles/wazuh/ansible-wazuh-manager : Installing the local_decoder.xml --------------------------------------------- 0.55s
../roles/wazuh/ansible-wazuh-manager : Adding local decoders files -------------------------------------------------- 0.53s
../roles/wazuh/ansible-wazuh-manager : Adding local rules files ----------------------------------------------------- 0.53s
../roles/wazuh/ansible-wazuh-manager : Check if NodeJS service exists ----------------------------------------------- 0.49s
../roles/wazuh/ansible-wazuh-manager : RedHat/CentOS/Fedora | Install Wazuh repo ------------------------------------ 0.37s

```